Test.Utility/TestEnvironmentBuilder.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
```

Comment:
Are you sure you need all these imports?

Reply:
Addressed in https://github.com/messerli-informatik-ag/test-utility/pull/11/commits/c8d6ab46905a494bd91a564b0e4a71f90a2ae690